frontend/src/employee-frontend/components/child-information/assistance/AssistanceFactorRow.tsx

```diff
@@ -47,6 +48,22 @@ export const AssistanceFactorRow = React.memo(function AssistanceFactorRow({
           })}
         />
       </Td>
+      <Td data-qa="modified-by">
+        {data.modified ? (
+          data.modifiedBy ? (
+            <Tooltip
+              tooltip={t.fields.lastModifiedBy(data.modifiedBy)}
+              position="left"
+            >
+              {data.modified.format()}
+            </Tooltip>
+          ) : (
+            data.modified.format()
+          )
+        ) : (
+          t.unknown
+        )}
+      </Td>
```

Comment:
This is fine too, but if I remember right you can pass an empty string / undefined to `Tooltip tooltip` prop in which case it just skips the tooltip and simply renders the children. Using that you could probably simplify this and reduce duplication